DOC: update the image name for docker image (#6143)
aiidateam/aiida-core: docs/source/intro/run_docker.rst

Run AiiDA via a Docker image
****************************

The AiiDA team maintains a `Docker <https://www.docker.com/>`__ image on `Docker Hub <https://hub.docker.com/r/aiidateam/aiida-core-with-services>`__.
This image contains a fully pre-configured AiiDA environment which makes it particularly useful for learning and testing purposes.

.. caution::

>     All data stored in a container will persist only over the lifetime of that particular container (which means remove the container will also purge the data) unless you use volumes (see instructions below).

Reviewer: ```suggestion
    All data stored in a container will persist only over the lifetime of that particular container (i.e., removing the container will also purge the data) unless you use volumes (see instructions below).
```

      To install Docker, please refer to the `official documentation <https://docs.docker.com/get-docker/>`__.

      .. note::

>          If you are using Linux, you need to have root privileges to do `post-installation steps for the Docker Engine <https://docs.docker.com/engine/install/linux-postinstall/>`__.

Author: This is not required anymore since for the linux the official docker doc also recommend to use docker desktop.

Reviewer: I think it is fine to have the latest docs to point to a release that does not exist yet. We will be releasing soon.
